internal/veracode-api-wrapper/updatemitigationinfo/README.md

```diff
@@ -0,0 +1,48 @@
+**Veracode API Wrapper - Update Mitigation Info**
```

Comment:
Let's use headings (#) and sub-headings (##) here. 
```suggestion
# Veracode API Wrapper - Update Mitigation Info
```

Reply:
Resolved